sdk/python/feast/feature_store.py

```diff
@@ -998,6 +999,48 @@ def serve(self, port: int) -> None:
 
         feature_server.start_server(self, port)
 
+    @log_exceptions_and_usage
+    def upload_docker_image(self) -> None:
```

Comment:
Why is this a top-level public method? I think this logic should live as part of update_infra in AwsProvider.